drivers/hmis/spec/system/hmis/mci_clearance_spec.rb

```diff
@@ -0,0 +1,262 @@
+require 'rails_helper'
+require_relative '../../requests/hmis/login_and_permissions'
+require_relative '../../support/hmis_base_setup'
+
+RSpec.feature 'Assessment definition selection', type: :system do
+  include_context 'hmis base setup'
+
+  before(:all) do
+    ::HmisUtil::JsonForms.new(env_key: 'allegheny').seed_record_form_definitions
+  end
```

Comment:
this will persist outside of the transaction; could lead to side-effects in other tests? Perhaps this should be `before(:each)`?